waiter/integration/waiter/basic_test.clj

```diff
@@ -180,11 +180,12 @@
 (deftest ^:parallel ^:integration-fast test-basic-unsupported-command-type
   (testing-using-waiter-url
     (let [headers {:x-waiter-name (rand-name)
+                   :x-waiter-version "1"
+                   :x-waiter-cmd "false"
```

Comment:
Why did we need to add these explicitly?

Reply:
You don't get a cmd-type error unless you have an otherwise properly defined service.